2-mandatory/6-journey-planner.js

```diff
@@ -81,7 +85,13 @@ function getTransportModes() {}
 
   Hint: Use the corresponding array method to decide if an element is included in an array.
 */
-function isAccessibleByTransportMode() {}
+function isAccessibleByTransportMode(transportArr, transport) {
+  if (transportArr.includes(transport)) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
for these we can simplify it by just returning the boolean straight from the result of the `includes` function

```suggestion
function isAccessibleByTransportMode(transportArr, transport) {
  return transportArr.includes(transport)
}
```